src/AboutUs/View/page.twig

```diff
@@ -3,7 +3,40 @@
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>O nas</h1>
+		<h1 class="u-center-text">Krótki opis naszego zespołu T</h1><br><br><br>
+		<p><strong>Daria Stasiak</strong>- młoda, prężna i niebywale głodna wiedzy. Absolutny ekspert w zaawansowanej technice ”ctrl C ctrl V”. Podjęła bardzo odważną decyzję o zmianie pracy i aktualnie stoi u progu świetlanej kariery jako programistka. Jej pasją jest taniec. <br><br>
```

Comment:
I recommend to use paragraph for each person. The br tag must be used only for line breaks that are actually part of the content, as in poems or addresses. It should not be used for separating thematic groups in a paragraph. Delete space between closing/opan tags and context. Example:
```<h3>Anna Sakławska</h3>```